scripts/install.sh

```diff
@@ -2,6 +2,7 @@
 
 set -e
 set -E # cause 'trap funcname ERR' to be inherited by child commands, see https://stackoverflow.com/questions/35800082/how-to-trap-err-when-using-set-e-in-bash
+set -o pipefail # Ensure the error is propagated past the piped commands
```

Comment:
@banjoh we cannot use this option
It will make stop to output the data when we run the script.

So, we will need to figure out a way to output and finish with the error.

Reply:
Removed